src/black/strings.py

```diff
@@ -18,6 +20,14 @@
     r"^([" + STRING_PREFIX_CHARS + r"]*)(.*)$", re.DOTALL
 )
 FIRST_NON_WHITESPACE_RE: Final = re.compile(r"\s*\t+\s*(\S)")
+UNICODE_RE = re.compile(
+    r"(\\+)("
+    r"(u([a-zA-Z0-9]{4}))"  # Formatting 16-bit unicodes i.e. \uxxxx
```

Comment:
Can remove the word "formatting", and "unicodes" isn't a thing, just say "characters" like https://docs.python.org/3.10/reference/lexical_analysis.html#string-and-bytes-literals